pbt-gen/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "pbt-gen"
+version = "0.1.0"
+authors = ["Shon Feder <shon.feder@gmail.com>"]
```

Comment:
Would you prefer to use your Gmail address over your Informal address here?

Reply:
Ah no, that was an oversight due to the template generation. Thanks.